### DISCUSSION TITLE
No updates since August 2023?

### DISCUSSION THREAD

**Discussion by fred8615** [Open-Shell/Open-Shell-Menu#1895] [General]
I noticed that there haven't been any updates to Open Shell since August 16th, 2023. Is this just an extended break, or has the program been abandoned?

**Comment by ge0rdi** [comment]
Not sure what you mean by _abandoned_.
Have a look at commits, issues, discussions, you will see that maintainers of the project are responding. If the project was abandoned, you won't hear anything from maintainers at all.

If you mean that there is no _active_ development, then that's true. Unfortunately there is nobody here that has enough free time (or skill) to work on new features (especially as MS isn't making 3rd party customization quite easy in recent Windows).

So I'd say the project is just being _maintained_.
We are trying to fix breaking issues, so that people can use `Open-Shell` on new Windows.

You can discuss/submit your own contributions if you are willing to improve some things.

**Reply by MadDog3577** [reply]
If I were a "coder", I'd jump in to help but I don't have that skill.  With that said, I understand the challenges in trying to keep up with operating system changes without a dedicated staff.

Q:  Is anyone reviewing the code of the current version of Open-Shell with a focus on security vulnerabilities?  In other words, is the code secure or is that an open question ATT?

Thanks for all you folks do to keep Open-Shell operational !!!

**Comment by fred8615** [comment]
Unfortunately I'm not a programmer. Just a user.

**Comment by NK2U** [comment]
Me too.  Would love to see a new version of Open Shell!
